sql/chunk.sql

```diff
@@ -65,6 +65,57 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.dimension_calculate_default_range_closed(
+        dimension_value   BIGINT,
+        num_slices        SMALLINT,
+    OUT range_start  BIGINT,
+    OUT range_end    BIGINT)
+LANGUAGE PLPGSQL STABLE AS
+$BODY$
+DECLARE
+    inter BIGINT;
+BEGIN
+       IF dimension_value < 0 THEN 
+            RAISE 'Dimension values for close dimensions should be positive. Got: %', dimension_value;
+       END IF;
+       inter := (2147483647 / num_slices);
+       IF dimension_value >= inter * (num_slices - 1) THEN
+          --put overflow from integer-division errors in last range
+          range_start = inter * (num_slices - 1);
+          range_end = 2147483647;
```

Comment:
Yeah there are no real constants but I made this nicer by making a param with the right default value